contrib/devtools/utxo_snapshot.sh

```diff
@@ -26,6 +28,57 @@ OUTPUT_PATH="${1}"; shift;
 # Most of the calls we make take a while to run, so pad with a lengthy timeout.
 BITCOIN_CLI_CALL="${*} -rpcclienttimeout=9999999"
 
+# Check if the node is pruned and get the pruned block height
+PRUNED=$( ${BITCOIN_CLI_CALL} getblockchaininfo | grep 'pruneheight' | awk '{print $2}' | tr -d ',' )
```

Comment:
On macOS 13.6 this silently exists with `-1` when a node is _not_ pruned (`pruneheight` is not set).

Reply:
I'll update it with an alternative, as I don't have an instance with macOS, please let me know if that would work better now.

Reply:
Done, pls let me know.